src/execution/operator/aggregate/physical_simple_aggregate.cpp

```diff
@@ -171,6 +177,8 @@ void PhysicalSimpleAggregate::Combine(ExecutionContext &context, GlobalSinkState
 
 		AggregateInputData aggr_input_data(aggregate.bind_info.get());
 		aggregate.function.combine(source_state, dest_state, aggr_input_data, 1);
+
+		gstate.state.counts[aggr_idx] += source.state.counts[aggr_idx];
```

Comment:
Perhaps `#ifdef DEBUG` this line?

Reply:
Good idea. I `#ifdef DEBUG`'d all lines that pertain to the verification